my-app/src/components/beachspeech/BSGameLobby.js

```diff
@@ -0,0 +1,52 @@
+import React from "react"
+import "../../stylesheets/beachspeech.css"
+import Island from "./Island"
+import CenterColumn from "./CenterColumn"
+import RightColumn from "./RightColumn"
+import Developers from "./Developers"
+import playerData from "../../data/beachspeech/playerData.json"
+
+/* Beach Speech Game Lobby */
+class BSGameLobby extends React.Component {
+    constructor() {
+        super()
+        this.state = {
+            gameCode: "ABCD",
+            id: "1234567890",
+            playersBS: playerData
```

Comment:
idk if we should put gameCode in state that would imply that it could be changed or will be changed would it make more sense if it was a const variable that is generated when a game is created? Im not too familiar with react so I might be a misunderstanding.

Reply:
Yeah, that makes sense, it should be a constant variable instead of a state.